scripts/core/EXT.rst

```diff
@@ -76,7 +103,7 @@ Etors must still use extension naming convention and values should be assigned t
 Extending Structures
 --------------------
 
-Any structure derived from ${x}_base_desc_t or ${x}_base_properties_t can be extended using structure chains.
+Any structure derived from base_desc_t or base_properties_t can be extended using structure chains.
```

Comment:
"Any structure derived from base descriptor or base property structure types may be extended using structure chains."

Reply:
fixed